Controller/Payconiq/Process.php

```diff
@@ -67,7 +70,10 @@ public function __construct(
         TransactionRepositoryInterface $transactionRepository,
         Account $account,
         TransactionCancel $transactionCancel,
-        QuoteRecreate $quoteRecreate
+        QuoteRecreate $quoteRecreate,
+        \Buckaroo\Magento2\Model\SecondChanceRepository $secondChanceRepository,
+        \Magento\Framework\Math\Random $mathRandom,
```

Comment:
not used I assume, can be removed

Reply:
Sorry, hover SecondChanceRepository and QuoteRecreate used, mathRandom and DateTime - removed